packages/web-client/app/components/card-pay/deposit-workflow/transaction-status/index.ts

```diff
@@ -56,7 +56,11 @@ class CardPayDepositWorkflowTransactionStatusComponent extends Component<Workflo
 
   constructor(owner: unknown, args: WorkflowCardComponentArgs) {
     super(owner, args);
-    if (!this.completedLayer2TxnReceipt) {
+    if (this.completedLayer2TxnReceipt) {
+      this.completedStepCount = 3;
```

Comment:
This is outside the scope of this PR but I hope we can find a way to automate this kind of thing, it’s always worrisome to me to have to manually intervene like this and only find out because of manual testing 😳

Reply:
The general approach is that restorable card state should all be derived from workflowSession state. If it is, there should be no need for this kind of procedural code.